src/main/java/com/keeper/homepage/domain/election/application/AdminElectionService.java

```diff
@@ -61,4 +73,47 @@ public Page<ElectionResponse> getElections(Pageable pageable) {
         .map(ElectionResponse::from);
   }
 
+  @Transactional
+  public void registerCandidate(String description, Long memberJobId, long electionId, long candidateId) {
+    Election election = validElectionFindService.findById(electionId);
+    Member member = memberFindService.findById(candidateId);
+    MemberJob memberJob = validCandidateService.findJobById(memberJobId);
+
+    ElectionCandidate electionCandidate = ElectionCandidate.builder()
+        .election(election)
+        .member(member)
+        .memberJob(memberJob)
+        .description(description)
+        .build();
+
+    electionCandidateRepository.save(electionCandidate);
+  }
+
+  @Transactional
+  public void registerCandidates(List<Long> candidateIds, String description, Long memberJobId, long electionId) {
+    Election election = validElectionFindService.findById(electionId);
+
+    for (Long candidateId : candidateIds) {
+      Member member = memberFindService.findById(candidateId);
+      MemberJob memberJob = validCandidateService.findJobById(memberJobId);
+
+      ElectionCandidate electionCandidate = ElectionCandidate.builder()
+          .election(election)
+          .member(member)
+          .memberJob(memberJob)
+          .description(description)
+          .build();
+
+      electionCandidateRepository.save(electionCandidate);
+    }
```

Comment:
위의 `registerCandidate`를 이용해서 아래처럼 재사용 할 수 있을 것 같네요~

그런데 1명 등록과 여러명 등록 API가 두 개 필요한 이유가 있을까요? 여러명 API로 1명만 등록해도 될 것 같아서요

```suggestion
    for (Long candidateId : candidateIds) {
      registerCandidate(description, memberJobId, electionId, candidateId);
    }
```

Reply:
저도 여러명 api로 1명만 등록하는 것도 괜찮다고 생각합니다! 오늘 회의 때 말씀드리고 수정하겠습니다~!

Reply:
이 부분은 지금 만들어둔 1명 등록 api는 그냥 남겨두고, 다음부터 1명 등록 api는 만들지 않는 것으로 이야기했습니다!